xikolo-common/Data/Model/Course.swift

```diff
@@ -80,6 +84,10 @@ class CourseSpine : BaseModelSpine {
             "language": Attribute(),
             "start_at": DateAttribute(),
             "end_at": DateAttribute(),
+            "status": Attribute(),
+            "hidden": Attribute(),
+            "open": Attribute(),
+            "external": Attribute(),
```

Comment:
`hidden`, `open` and `external` are bools. Please use `xx_int` and add a boolean setter/getter.